axelar-chains-config/info/stagenet.json

```diff
@@ -2509,6 +2509,28 @@
           "initializeArgs": {}
```

Comment:
PR title can be a bit more descriptive
`chore(evm): add celo-sepolia to stagenet / testnet`